scipy/signal/__init__.py

```diff
@@ -294,7 +294,7 @@
 from ._max_len_seq import max_len_seq
 from ._upfirdn import upfirdn
 
-from ._spline import (
+from ._spline import (  # noqa: F401
```

Comment:
This can be avoided. It is ignored in `tox.ini`.

Reply:
It isn't ignored by `lint_diff.py` though.

Reply:
I see. We still are going to check for all PRs.